client/client.go

```diff
@@ -81,6 +82,10 @@ type Client struct {
 	manualOverride bool
 }
 
+func redirectError(req *http.Request, via []*http.Request) error {
+	return errors.New("Sorry daemon could not process your request. Please make sure you entered correct arguments")
```

Comment:
Here's the current behavior:

```
docker inspect ../asdf
[]
Error: No such object: ../asdf
```

So that seems reasonable as long as we keep that error.